components/builder-api/src/server/resources/pkgs.rs

```diff
@@ -176,7 +177,7 @@ fn get_packages_for_origin(req: HttpRequest,
     let origin = path.into_inner();
     let ident = PackageIdent::new(origin, String::from(""), None, None);
 
-    match do_get_packages(&req, &ident, &pagination) {
+    match do_get_packages(&req, &ident, &pagination, false) {
```

Comment:
rather than adding a `fetchall` param. Should we assume all packages if `range` == `-1`?